src/Swarm/TUI/View.hs

```diff
@@ -490,9 +490,7 @@ drawConst :: Const -> Widget Name
 drawConst c = hBox [padLeft (Pad $ 13 - T.length constName) (txt constName), txt constSig]
  where
   constName = syntax . constInfo $ c
-  constSig = case processParsedTerm (noLoc $ TConst c) of
-    Right (ProcessedTerm (TConst _) (Module pt _) _ _) -> " : " <> prettyText pt
-    _ -> "??"
+  constSig = " : " <> prettyText (inferConst c)
```

Comment:
@TristanCacqueray Hah, I see you had fun drawing the `Const` type too! :grinning: